app/forms/state_file/nj_college_dependents_exemption_form.rb

```diff
@@ -0,0 +1,17 @@
+module StateFile
+  class NjCollegeDependentsExemptionForm < QuestionsForm
+    attr_accessor :dependents_attributes
+    delegate :dependents, to: :intake
+
+    set_attributes_for(:intake)
```

Comment:
[sand] Without a second argument, does this do anything? I think it will just create a `nil` entry in the scoped attributes hash.

Reply:
Good, catch, you're right. Will remove.